src/dev-box/dev-box-k8s-deploy.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  annotations:
+    prometheus.io/scrape: 'true'
+  name: dev-box
+spec:
+  selector:
+    matchLabels:
+      app: dev-box
+  template:
+    metadata:
+      labels:
+        app: dev-box
+      name: dev-box
+    spec:
+      nodeSelector:
+        dev-box: "true"
+      containers:
+      - image: docker.io/openpai/dev-box
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /root/.docker
+          name: docker-cred-volume
+        - mountPath: /bin/docker
```

Comment:
Can you ensure the binary will always be here?

Reply:
will remove and keep  -v /var/run/docker.sock:/var/run/docker.sock \